packages/exchange/src/exchange/langfuse_wrapper.py

```diff
@@ -48,7 +48,7 @@ def auth_check() -> bool:
 CURRENT_DIR = Path(__file__).parent
 PACKAGE_ROOT = find_package_root(CURRENT_DIR)
 
-LANGFUSE_ENV_FILE = os.path.join(PACKAGE_ROOT, ".env.langfuse.local")
+LANGFUSE_ENV_FILE = os.path.join(PACKAGE_ROOT, ".env.langfuse.local1") if PACKAGE_ROOT else None
```

Comment:
did you mean to change it to local1 @lifeizhou-ap

Reply:
oops! sorry I was using the name for testing

Reply:
thought so! good catch